nupic/regions/TPRegion.py

```diff
@@ -506,6 +517,17 @@ def _compute(self, inputs, outputs):
       size = activeLearnCells.shape[0] * activeLearnCells.shape[1]
       outputs['lrnActiveStateT'][:] = activeLearnCells.reshape(size)
 
+    computePredictedActiveCellIndices = getattr(self, "computePredictedActiveCellIndices", False)
```

Comment:
Is this for backwards compatibility? It might be better to make sure this attribute gets set in the `__setstate__` method and then elsewhere in the code you can assume it is set.